Adjust copyright notice.

jakevdp opened the pull request:
Previously we had been pulling-in NumPy and SciPy docs at runtime, but after the work in #21461 this is no longer the case.

hawkinsp commented on docs/conf.py:
```diff
@@ -49,7 +49,7 @@ def _do_not_evaluate_in_jax(
 
 project = 'JAX'
-copyright = '2024, The JAX Authors. NumPy and SciPy documentation are copyright the respective authors.'
+copyright = '2024, The JAX Authors.'
```
Do we still end up with a trailing double period?

jakevdp replied:
Fixed, thanks!

jakevdp committed:
Adjust copyright notice.

Previously we had been pulling-in NumPy and SciPy docs at runtime, but
after the work in #21461 this is no longer the case.
